pkg/bootstrap/config.go

```diff
@@ -619,6 +630,39 @@ func GetNodeMetaData(options MetadataOptions) (*model.Node, error) {
 	}, nil
 }
 
+func GetNodeMetaDataLabels(platform platform.Environment) (map[string]string, error) {
```

Comment:
can we reuse this within `GetNodeMetaData` to avoid code drift?

Reply:
If so, i would rather make use of `func (a *Agent) generateNodeMetadata() `